experimental/traffic-portal/src/app/core/statuses/status-details/status-details.component.html

```diff
@@ -0,0 +1,45 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<mat-card class="statuses-details">
+    <tp-loading *ngIf="loading"></tp-loading>
+    <form [formGroup]="statusDetailsForm" (ngSubmit)="onSubmit($event)" *ngIf="!loading">
+        <mat-card-content>
+            <mat-form-field *ngIf="!new">
+                <mat-label>ID</mat-label>
+                <input matInput type="text" name="id" disabled readonly [defaultValue]="statusDetails.id" />
+            </mat-form-field>
+            <mat-form-field>
+                <mat-label>Name</mat-label>
+                <input matInput type="text" name="name" required formControlName="name">
+            </mat-form-field>
+            <mat-form-field>
+                <mat-label>Description</mat-label>
+                <input matInput type="text" name="description" formControlName="description">
+            </mat-form-field>
+            <mat-form-field *ngIf="!new">
+                <mat-label>Last Updated</mat-label>
+                <input matInput type="text" name="lastUpdated" disabled readonly
+                    [defaultValue]="statusDetails.lastUpdated" />
+            </mat-form-field>
+        </mat-card-content>
+        <mat-card-actions align="end">
+            <button mat-raised-button type="button" color="warn" (click)="deleteStatus()" *ngIf="!new">Delete</button>
+            <button mat-raised-button type="button" routerLink="/core/statuses">Cancel</button>
```

Comment:
Buttons should never have `routerLink` attributes. But also, this button doesn't need to exist, since this navigation is available from the menu on the left.